### PR TITLE
updated stale links

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-task-update-postman-oas-links.s3-website-us-east-1.amazonaws.com/index.html

**Review by ajrice6713 — APPROVED** [review]

File: SUMMARY.md [hunk 1]
```diff
@@ -18,9 +18,9 @@
 		* [Voice, Usage and Migration GUide](sdks/node/usage-guide-voice.md)
 	* [Java](sdks/java.md)
 	* [PHP](sdks/php.md)
-	* [Examples on Github](https://github.com/Bandwidth/examples)
-	* [OpenAPI/Swagger on Github](https://github.com/Bandwidth/examples/tree/master/public-specs)
-    * [Postman Collections](https://github.com/Bandwidth/examples/tree/master/postman-collections)
+	* [Examples on Github](https://github.com/bandwidth-samples)
+	* [OpenAPI/Swagger on Github](https://github.com/Bandwidth/openapi-specs)
```
Comment (hamermike) [inline SUMMARY.md]:
```suggestion
	* [Examples on GitHub](https://github.com/bandwidth-samples)
	* [OpenAPI/Swagger on GitHub](https://github.com/Bandwidth/openapi-specs)
```

File: sdks/about.md [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 | Link                                                                                        | Description                                                                                        |
 |:--------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
-| [Examples on Github](https://github.com/Bandwidth/examples)                                 | Example applications & Scripts demo-ing Bandwidth functionality with the SDKs                      |
-| [OpenAPI/Swagger on Github](https://github.com/Bandwidth/examples/tree/master/public-specs) | Bandwidth's [Open API Specification](https://swagger.io/solutions/getting-started-with-oas/) files |
+| [Examples on Github](https://github.com/bandwidth-samples)                                 | Example applications & Scripts demo-ing Bandwidth functionality with the SDKs                      |
+| [OpenAPI/Swagger on Github](https://github.com/Bandwidth/openapi-specs) | Bandwidth's [Open API Specification](https://swagger.io/solutions/getting-started-with-oas/) files |
```
Comment (hamermike) [inline sdks/about.md]:
```suggestion
| [Examples on GitHub](https://github.com/bandwidth-samples)                                 | Example applications & Scripts demo-ing Bandwidth functionality with the SDKs                      |
| [OpenAPI/Swagger on GitHub](https://github.com/Bandwidth/openapi-specs) | Bandwidth's [Open API Specification](https://swagger.io/solutions/getting-started-with-oas/) files |
```

**Review by hamermike — CHANGES_REQUESTED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-task-update-postman-oas-links.s3-website-us-east-1.amazonaws.com/index.html

**Review by hamermike — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-task-update-postman-oas-links.s3-website-us-east-1.amazonaws.com/index.html

**Pull request merged**